server/job_controller/api_services/audio_process.py

```diff
@@ -9,12 +9,24 @@
 from google.cloud.speech import enums
 from google.cloud.speech import types
 
+
+# Set up Google Custom Search for identifying movies
+from dotenv import load_dotenv
+import requests
+import collections
+
+load_dotenv()
+API_KEY = os.getenv('CUSTOM_SEARCH_API_KEY')
+CSE_ID = os.getenv('CUSTOM_SEARCH_ID')
+
+base_url = "https://www.googleapis.com/customsearch/v1"
+
 def ExtractAudio(video_path):
-    outputPath = "../audio/Inception_Audio.mp3"
+    fileName = re.search("(.*\/|^)(.*?)\.mp4", video_path).group(2)
```

Comment:
can you replace this regex with something simpler? like (.*)\.mp4 and capture group 1 instead of 2? looks like this regex is capturing a group and ignoring it?